src/click/core.py

```diff
@@ -1193,8 +1193,10 @@ def get_short_help_str(self, limit=45):
         """Gets short help for the command or makes it by shortening the
         long help string.
         """
+        short_help_text = DEPRECATED_HELP_NOTICE + self.short_help
```

Comment:
Should only be appended if the command is deprecated.

Reply:
Yes I overlooked that, updated the code.